src/Api/Controllers/SsoUserController.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Bit.Core.Repositories;
+using Bit.Core.Models.Table;
+using Bit.Core.Exceptions;
+using Bit.Core.Services;
+
+namespace Bit.Api.Controllers
+{
+    [Route("sso-user")]
+    public class SsoUserController : Controller
+    {
+        private readonly IUserService _userService;
+        private readonly ISsoUserRepository _ssoUserRepository;
+
+        public SsoUserController(IUserService userService, ISsoUserRepository ssoUserRepository)
+        {
+            _userService = userService;
+            _ssoUserRepository = ssoUserRepository;
+        }
+
+        [HttpDelete("{organizationId}")]
+        public async Task Delete(string organizationId)
```

Comment:
let's add this to the existing `AccountsController` as `DeleteSsoUser` or something to that tune and remove the new `SsoUserController`.

Reply:
Yes, lest just create `DELETE /accounts/sso/orgId` or something